docs/usage.md

```diff
@@ -166,8 +172,21 @@ are also required:
 -Wl,--wrap=main
 -Wl,--dynamic-list=$IA2_PATH/runtime/libia2/dynsym.syms
 -Wl,--export-dynamic
+-L$IA2_PATH/build/runtime/libia2
+-L$IA2_PATH/build/runtime/partition-alloc
+-llibia2
+-lpartition-alloc
+-lcallgates
 ```
 
 Also if the rewriter produces a linker args file for a given compartment (i.e. a
 `.ld` file), you must include `-Wl,@/path/to/generated_linker_args_$PKEY.ld` when
 linking that DSO.
+
+## Using Thread Local Storage
+
+When using TLS in a containerized app, you'll need to run the `pad-tls` tool on
+all shared objects in the app. This includes `libc.so` as libc both initializes
```

Comment:
Maybe mention `ldd` and `lddtree` as ways to list all DSOs a program uses.

Reply:
Good idea, added.